.github/workflows/chatbot-finetune-mpt-7b-chat.yml

```diff
@@ -19,39 +19,41 @@ jobs:
         run: cat ~/itrex-actions-runner/.env >> $GITHUB_ENV
 
       - name: Build Docker Image
-        run: docker build ./ --target cpu --build-arg http_proxy="${{ env.HTTP_PROXY_IMAGE_BUILD }}" --build-arg https_proxy="${{ env.HTTPS_PROXY_IMAGE_BUILD }}" -f workflows/chatbot/fine_tuning/docker/Dockerfile -t chatbotfinetune:latest && yes | docker container prune && yes | docker image prune
+        run: docker build ./ --target cpu --build-arg http_proxy="${{ env.HTTP_PROXY_IMAGE_BUILD }}" --build-arg https_proxy="${{ env.HTTPS_PROXY_IMAGE_BUILD }}" -f workflows/chatbot/fine_tuning/docker/Dockerfile -t chatbotfinetune-mpi:latest && yes | docker container prune && yes | docker image prune
```

Comment:
Is that possible to setup proxy info in ~/docker.config on the server, instead of passing to docker as params?

Reply:
yes, it's possible. But it's more dynamic to pass it as parameter since we don't always have permission to change docker daemon in all environments. We have many self-hosted envs. It's also for distinguishing proxies between docker build and docker run since they use different networks.